src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -315,11 +316,23 @@ export default defineComponent({
   },
   mounted: function () {
     const volume = sessionStorage.getItem('volume')
+    const muted = sessionStorage.getItem('muted')
 
     if (volume !== null) {
       this.volume = volume
     }
 
+    if (muted !== null) {
+      // as sessionStorage stores string values which are truthy by default so we must check with 'true'
+      // otherwise 'false' will be returned as true as well
+      this.muted = (muted === 'true')
+    }
+
+    if (muted === 'false' && volume === '0') {
+      this.muted = true
+      sessionStorage.setItem('volume', 0.25)
```

Comment:
Why is there a hardcoded value of 0.25 here?

Reply:
Oh frick, i was testing smth didn't remove it. Sorry!!